### PR TITLE
Feat/design the landing page - MAILPOET-4799

### PR DESCRIPTION
**Pull request by triple0t**
## Description
This PR adds the final content for the landing page.

mailpoet-landingpage

## QA notes

You can access the page here: `wp-admin/admin.php?page=mailpoet-landingpage`

You need to enable Landingpage experimental features here: `/wp-admin/admin.php?page=mailpoet-experimental`

Check https://github.com/mailpoet/mailpoet/pull/4620 for more QA information. 

When testing using the ZIP build, the images on the page won't load because they will be publicly available after the release.

You can check the feature with the images by testing with the dev env and adding `define( 'MAILPOET_USE_CDN', false ); ` to the wp-config.php. Alternatively, you can ping me, and I'll push the images to the repo so that they also work from the zip.

## Linked tickets

[MAILPOET-4799](https://mailpoet.atlassian.net/browse/MAILPOET-4799)

[MAILPOET-4799]: https://mailpoet.atlassian.net/browse/MAILPOET-4799?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: mailpoet/assets/css/src/components-plugin/_landingpage.scss [hunk 1]
```diff
@@ -136,6 +150,12 @@
         transform-origin: center;
         transition: .2s transform ease;
       }
+
+      @include respond-to(small-screen) {
+        &:after {
+          right: 0;
+        }
```
Comment (brezocordero) [inline mailpoet/assets/css/src/components-plugin/_landingpage.scss:172]: On the mobile view, the summary in some of the FAQs is very close to the arrow. Maybe increase the summary right padding a bit?

File: mailpoet/assets/js/src/landingpage/content.tsx [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+};
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+const Images: LandingpageImages = window.mailpoet_landingpage_images;
+
+function Content() {
+  return (
+    <div className="landing-content">
+      <div className="hero-section mailpoet-content-center">
```
Comment (brezocordero) [inline mailpoet/assets/js/src/landingpage/content.tsx:23]: @triple0t what do you think about using some semantic HTML? like sections for the hero, general features and FAQs? Maybe also lists on the features?

File: mailpoet/assets/css/src/components-plugin/_landingpage.scss [hunk 3]
```diff
@@ -30,6 +30,90 @@
...
+    .landingpage-wooCommerce-features {
+      margin-top: 30px;
+      padding: 2rem 10rem;
+
+      .landingpage-wooCommerce-feature-item {
+        padding: 30px;
+
+        @media screen and (min-width: 960px) and (max-width: 1390px) {
+          .landingpage-images {
+            width: 100%;
```
Comment (brezocordero) [inline mailpoet/assets/css/src/components-plugin/_landingpage.scss:111]: Small thing, but this is how it displays on my M1 13in, the text overlaps a bit with the image. 
Screen Shot 2023-01-13 at 4 38 23 PM
A bit smaller or bigger screen and there is no overlap.

**Review by brezocordero — COMMENTED** [review]
@triple0t Great work.
I just have some smaller comments but nothing blocking.

**Comment by triple0t** [comment]
Thank you for the review, @brezocordero 

Issues highlighted have been fixed in https://github.com/mailpoet/mailpoet/pull/4651/commits/d04b72e18da1014446f47ad247937062a8153515

**Review by brezocordero — APPROVED** [review]
Looks good! Thanks for the changes.

**Review by Aschepikov — APPROVED** [review]
GJ!

**Pull request merged**